R/prepare_istr_product.R

```diff
@@ -46,7 +53,8 @@ rename_istr_product <- function(data) {
       ep_product = "clustered",
       ISTR_risk_category = "risk_category",
       input_name = "exchange_name",
-      input_unit = "exchange_unit_name"
+      input_unit = "exchange_unit_name",
+      input_isic_4digit_name = "isic_4digit_name_ecoinvent"
```

Comment:
NEWS:

* The output of `profile_sector_upstream()` at product level gains the column `input_isic_4digit_name`.